src/cljs_bean/core.cljs

```diff
@@ -222,3 +241,13 @@
        (true? keywordize-keys) (bean x)
        (and (some? prop->key) (some? key->prop)) (Bean. nil x prop->key key->prop nil)
        :else (throw (js/Error ":keywordize-keys or both :prop->key and :key->prop must be specified"))))))
+
+(defn bean?
+  "Returns true if x is a bean."
+  [x]
+  (instance? Bean x))
+
+(defn object
+  "Takes a bean and returns a JavaScript object."
+  [bean]
+  (gobj/clone (.-obj bean)))
```

Comment:
The `gobj/clone` here is just trying to play things safe. That way client code can't use this to mutate the object held by `Bean`.